packages/prisma/index.ts

```diff
@@ -6,9 +6,16 @@ import { bookingReferenceMiddleware } from "./middleware";
 
 const prismaOptions: Prisma.PrismaClientOptions = {};
 
+const globalForPrisma = global as unknown as {
+  prismaWithoutClientExtensions: PrismaClientWithoutExtension;
+  prismaWithClientExtensions: PrismaClientWithExtensions;
+};
+
 if (!!process.env.NEXT_PUBLIC_DEBUG) prismaOptions.log = ["query", "error", "warn"];
 
-const prismaWithoutClientExtensions = new PrismaClientWithoutExtension(prismaOptions);
```

Comment:
This was invoking a new client each time this file is imported.

Introduced here:

https://github.com/calcom/cal.com/pull/10687/files#r1370735278